.github/workflows/superset-frontend.yml

```diff
@@ -36,11 +36,20 @@ jobs:
         uses: ./.github/actions/cached-dependencies
         with:
           run: npm-install
-      - name: lint
+      - name: eslint
```

Comment:
breaking down the steps in CI makes it more clear where the output is coming from, and gets timing info on the different steps. Helping understand the time it takes for each step